pubsub/streaming-analytics/requirements.txt

```diff
@@ -1 +1,2 @@
-apache-beam[gcp]==2.15.0
+apache-beam[gcp]==2.16.0
+nose==1.3.7
```

Comment:
Without specifying it would also work. I wanted to keep the version so at least we know which version definitely works in case diamond dependencies issues come up for people in the future and they need some version number for diagnosis.

Reply:
Is `nose` itself necessary here or is already installed by `nox`?

Reply:
It's not already installed by `nox`

```
___________________ ERROR collecting pubsub/streaming-analytics/PubSubToGCS_test.py ___________________
ImportError while importing test module '/Users/tianzi/Projects/python-docs-samples/pubsub/streaming-analytics/PubSubToGCS_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
PubSubToGCS_test.py:20: in <module>
    from apache_beam.testing.test_pipeline import TestPipeline
../../.nox/py36-sample-pubsub-streaming-analytics/lib/python3.6/site-packages/apache_beam/testing/test_pipeline.py:25: in <module>
    from nose.plugins.skip import SkipTest
E   ModuleNotFoundError: No module named 'nose'
```

Reply:
Got it, it looks like `TestPipeline` depends on `nose`.

Can you do `apache-beam[gcp,test]==2.16.0` instead? I think that should include `nose` and any other hidden dependency we might get into.

Reply:
Done. 👍